src/pilot/inspect.patch

```diff
@@ -0,0 +1,70 @@
+--- inspect.py	2019-03-18 04:26:06.117996201 -0400
++++ inspect_latest.py	2019-03-18 04:36:58.007653944 -0400
+@@ -119,10 +119,17 @@
+                       {'node_uuid': node.uuid, 'error': exc})
+             raise exception.HardwareInspectionFailure(error=exc)
+ 
++        pxe_dev_nics = self._get_pxe_dev_nics(client, nics, node)
++        if pxe_dev_nics is None:
++            LOG.warning(_LW('No PXE enabled NIC was found for node '
```

Comment:
The translators for log levels, such as `_LE()`, should not be present. They were removed during Queens and are not in the OSP13z5 version of `inspect.py`. You can see from the cherry pick that added PXE enabled to Queens that they are not there -- https://review.openstack.org/#/c/624388/2/ironic/drivers/modules/drac/inspect.py

Reply:
Yes, He did create patch based on Queens